.github/workflows/docker-image.yml

```diff
@@ -106,6 +106,39 @@ jobs:
       - name: ensure systemd container is gone
         run: sudo podman ps -a
 
+  kube-validate:
+    runs-on: ubuntu-latest
```

Comment:
ubuntu required?

Reply:
@jeffvance the depressing part about github actions is that they don't have any free centos/rhel/fedora instances for our testing